src/cli.nim

```diff
@@ -416,6 +453,16 @@ proc processCmdLine*: Conf =
     if result.action.offline and result.action.probSpecsDir.len == 0:
       showError(&"'{list(optSyncOffline)}' was given without passing " &
                 &"'{list(optSyncProbSpecsDir)}'")
+    if result.action.scope.len == 0:
+      result.action.scope = {SyncKind.low .. SyncKind.high}
+    if result.action.update and result.action.yes and skTests in result.action.scope:
+      let msg = fmt"""
+        '{list(optSyncYes)}' cannot be used when updating tests
+        You can either:
+        - remove  '{list(optSyncYes)}'
+        - or narrow the syncing scope via some combination of --docs, --filepaths, and --metadata
```

Comment:
Isn't there a third option where the user use the "mode" argument to specify what to do when encountering an unknown tests?

Reply:
Yup. I've tried to address this situation in https://github.com/exercism/configlet/pull/366/commits/95965e9a5ae85b894c6334e0d0000e5d9dd4949d.

I'll mark this as resolved and we can discuss further if necessary in the main PR body, since we have https://github.com/exercism/configlet/pull/366#issuecomment-966163378.